app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
```

Comment:
- [ ]  Please modify the registration view in devise to match the wireframes and their styling.
## Wireframe Guidelines
![image](https://user-images.githubusercontent.com/46839548/200141111-b5479235-651b-4854-8845-61036ec1dfe2.png)

## Your Implementation
![image](https://user-images.githubusercontent.com/46839548/200141171-3fa0b9c8-1d44-4833-a84a-3be2e8884f50.png)

- [ ] An error 🐛 occurs when registering a new user. As a result, I am unable to test the login, email confirmation, and password recovery functionality of your app. This error has to do with the `post_counter` column in your users table. Please in your migration, kindly assign it a default value of 0 in order to get rid of this error. Please try to fix it 🙏
![image](https://user-images.githubusercontent.com/46839548/200141775-2f673c4f-97f7-49ad-bb86-e4f1d86ac483.png)